chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/CertifiedProductDAO.java

```diff
@@ -307,6 +308,31 @@ public List<CertifiedProductDetailsDTO> getListingsByStatus(Collection<Certifica
                 .collect(Collectors.toList());
     }
 
+    @Transactional(readOnly = true)
+    public List<CertifiedProductDetailsDTO> getListingsByStatusExcludingEdition(
```

Comment:
It's weird for the method name to say "excluding edition", but the where clause to use "editions to exclude". Maybe the name could be `getListingsByStatusExcludingRetiredEditions`? I think we got confirmation that 2015 won't be Retired, so that would exclude 2011 and 2014 only

Reply:
Updated. Thank you for the suggestion.